KitePrintSDK/src/main/java/ly/kite/address/Country.java

```diff
@@ -1,343 +1,501 @@
-        COUNTRIES.add(new Country("Taiwan, Province of China", "TW", "TWN", "TWD"));
-        COUNTRIES.add(new Country("Tajikistan", "TJ", "TJK", "TJS"));
-        COUNTRIES.add(new Country("Tanzania, United Republic of", "TZ", "TZA", "TZS"));
-        COUNTRIES.add(new Country("Thailand", "TH", "THA", "THB"));
-        COUNTRIES.add(new Country("Timor-Leste", "TL", "TLS", "USD"));
-        COUNTRIES.add(new Country("Togo", "TG", "TGO", "XOF"));
-        COUNTRIES.add(new Country("Tokelau", "TK", "TKL", "NZD"));
-        COUNTRIES.add(new Country("Tonga", "TO", "TON", "TOP"));
-        COUNTRIES.add(new Country("Trinidad and Tobago", "TT", "TTO", "TTD"));
-        COUNTRIES.add(new Country("Tunisia", "TN", "TUN", "TND"));
-        COUNTRIES.add(new Country("Turkey", "TR", "TUR", "TRY"));
-        COUNTRIES.add(new Country("Turkmenistan", "TM", "TKM", "TMT"));
-        COUNTRIES.add(new Country("Turks and Caicos Islands", "TC", "TCA", "USD"));
-        COUNTRIES.add(new Country("Tuvalu", "TV", "TUV", "AUD"));
-        COUNTRIES.add(new Country("Uganda", "UG", "UGA", "UGX"));
-        COUNTRIES.add(new Country("Ukraine", "UA", "UKR", "UAH"));
-        COUNTRIES.add(new Country("United Arab Emirates", "AE", "ARE", "AED"));
-        COUNTRIES.add(new Country("United Kingdom", "GB", "GBR", "GBP"));
-        COUNTRIES.add(new Country("United States", "US", "USA", "USD"));
-        COUNTRIES.add(new Country("United States Minor Outlying Islands", "UM", "UMI", "USD"));
-        COUNTRIES.add(new Country("Uruguay", "UY", "URY", "UYU"));
-        COUNTRIES.add(new Country("Uzbekistan", "UZ", "UZB", "UZS"));
-        COUNTRIES.add(new Country("Vanuatu", "VU", "VUT", "VUV"));
-        COUNTRIES.add(new Country("Venezuela, Bolivarian Republic of", "VE", "VEN", "VEF"));
-        COUNTRIES.add(new Country("Viet Nam", "VN", "VNM", "VND"));
-        COUNTRIES.add(new Country("Virgin Islands, British", "VG", "VGB", "USD"));
-        COUNTRIES.add(new Country("Virgin Islands, U.S.", "VI", "VIR", "USD"));
-        COUNTRIES.add(new Country("Wallis and Futuna", "WF", "WLF", "XPF"));
-        COUNTRIES.add(new Country("Western Sahara", "EH", "ESH", "MAD"));
-        COUNTRIES.add(new Country("Yemen", "YE", "YEM", "YER"));
-        COUNTRIES.add(new Country("Zambia", "ZM", "ZMB", "ZMW"));
-        COUNTRIES.add(new Country("Zimbabwe", "ZW", "ZWE", "ZWL"));
-
-        // Quick sanity check to only add countries that have all the details we want
-        for (Country c : COUNTRIES) {
-            if (c.code2.trim().length() == 2 && c.code3.trim().length() == 3 && c.currencyCode.trim().length() > 0) {
-                Country.COUNTRIES.add(c);
-            }
-        }
+
+///// Class Declaration /////
+
+/*****************************************************
+ *
+ * This enum represents a country.
+ *
+ *****************************************************/
+public enum Country
+  {
+    ALAND_ISLANDS                     ( "Ã…land Islands", "AX", "ALA", "EUR", true ),
```

Comment:
Oh yuck, I think when I gave you the fresh country list `Åland Islands` got encoded incorrectly -- my bad. Could you replace `Ã…land Islands` with `Åland Islands`